config/routes.rb

```diff
@@ -1,8 +1,7 @@
 Rails.application.routes.draw do
   devise_for :admin_users, controllers: {
     sessions: 'admin_users/sessions',
-    registrations: 'admin_users/registrations',
-    passwords: 'admin_users/passwords'
+    registrations: 'admin_users/registrations'
   }
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
if you have changes model name then routes must be according to that model
devise_for :users